src/octoprint/settings.py

```diff
@@ -168,6 +168,7 @@ def settings(init=False, basedir=None, configfile=None, overlays=None):
         "resendRatioThreshold": 10,
         "resendRatioStart": 100,
         "ignoreEmptyPorts": False,
+        "encodingScheme": "ascii",
```

Comment:
I think just `encoding` would be better.

Reply:
You got it on both counts.  I didn't have sufficient context as to how this would be received and took the more restrictive path.

I'll amend with both requested changes this evening (US ET time).

Thank you for helping further my non-typical use case.